models/User.js

```diff
@@ -27,6 +27,14 @@ const UserSchema = new mongoose.Schema({
       "Please provide a valid password between 6 and 25 characters in length.",
     ],
     minLength: 6,
+    maxLength: 25,
+  },
+  team: {
```

Comment:
This is totally good as an implementation - other option you could see is a separate team model where the Id is linked here. But not required - only comment is if a user has a team the name is the only identifier, so you’ll likely want a list of teams shown on the FE so people don’t accidentally “create” a team with the same name as an existing one